.github/workflows/docker-tags.yml

```diff
@@ -38,4 +38,6 @@ jobs:
         file: ci/Dockerfile.dockerhub
         push: true
         tags: |
+          # TODO(bbannier): Automatically detect whether this is the latest
+          # release and in that case also push this to `spicy:latest`.
```

Comment:
Can we always push to latest: the most recently pushed release will always be the latest, won't it?

Reply:
Right now that is true, but once we have two releases out and do patch releases. We would need to compare the commit of the current tag against `release` or `beta` or something similar.

I don't think this is too big an issue for now, and we have it documented in our release checklist.